flixel/graphics/tile/FlxDrawTilesItem.hx

```diff
@@ -33,7 +33,7 @@ class FlxDrawTilesItem extends FlxDrawBaseItem<FlxDrawTilesItem>
 	}
 	
 	override public function addQuad(frame:FlxFrame, matrix:FlxMatrix,
-		red:Float = 1, green:Float = 1, blue:Float = 1, alpha:Float = 1):Void
+		red:Float = 1, green:Float = 1, blue:Float = 1, alpha:Float = 1, redOffset:Float = 0, greenOffset:Float = 0, blueOffset:Float = 0, alphaOffset:Float = 0):Void
```

Comment:
openfl drawtiles takes an array of floats, I suppose I could make them integers here but they would be converted to floats later